src/index.html

```diff
@@ -7,12 +7,75 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__block-img">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="imac"
+          width="160"
+          height="134"
+        />
+      </div>
+
+      <div class="product-card__name">
+        <h2 class="title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="product-card__product-code">
+        <p class="text">Product code: 195434</p>
+      </div>
+
+      <div class="product-card__block-stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="reviews">
```

Comment:
```suggestion
```

Reply:
Not fixed 
<img width="1063" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/fa591e17-5d5a-4391-8013-1415dcbb46f7">